src/platform/CMakeLists.txt

```diff
@@ -12,16 +12,23 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-add_library(platform STATIC
-  platform_linux.cpp
-  platform_shared.cpp
-  platform_unix.cpp)
 
-target_link_libraries(platform
-  qemu_backend
-  libvirt_backend
-  lxd_backend
-  journaldlogger)
+if(LINUX)
+  add_library(platform STATIC
+    platform_linux.cpp
+    platform_shared.cpp
+    platform_unix.cpp)
+
+  target_link_libraries(platform
+    libvirt_backend
+    lxd_backend
+    journaldlogger)
+endif()
```

Comment:
I did this to keep from having to fix conflicts in the private branch.  I asked on chat if there was agreement to do this to avoid the conflicts and all I got was "yes" :slightly_smiling_face:

Reply:
This is no longer applicable.